index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Landing Page</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <a href="#"
+        ><img class="img_1" src="images/food logo.jpg" alt="food logo"
+      /></a>
+      <nav>
+        <ul class="link_container">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="sect">
+      <div class="section_1">
+        <h1>NAIJA'S KITCHEN</h1>
+        <p>
+          African foods are bountiful and diverse. They are rich in nutritional
+          fiber and often unrefined and they offer a healthy choice when
+          consumed in the right blend.
+        </p>
+        <button class="btn_signup" type="button">sign up</button>
+      </div>
+      <div>
+        <img src="images/catering.jpg" alt="African foods" />
+      </div>
+    </section>
+    <section class="caterers">
+      <div>
+        <h2>These Are African Caterers</h2>
+        <ul class="caterers_list">
+          <li>
+            <div class="image_contains">
+              <img src="images/cater.jpg" alt="first caterer" />
+              <h3>Joy Paul</h3>
+              <p>Abia, expert in jollof rice</p>
+            </div>
+          </li>
+          <li>
+            <div class="image_contains">
+              <img src="images/caterers.jpg" alt="second caterer" />
+              <h3>Precious Izo</h3>
+              <p>Lagos, expert in Afang soup</p>
+            </div>
+          </li>
+          <li>
+            <div class="image_contains">
+              <img src="images/caterers3.jpg" alt="Third caterer" />
+              <h3>Nkechi Dan</h3>
+              <p>Rivers, awesome cook</p>
+            </div>
+          </li>
+          <li>
+            <div class="image_contains">
+              <img src="images/caterers4.jpg" alt="Fourth caterer" />
+              <h3>Peter Elvis</h3>
+              <p>Abuja, expert in edikanikon</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] The images in this section are way too small, you can look at the template and try to make it look alike. This is an essential skill to be able to replicate product requirements for your clients.  Follow the video I did, see how I used images that are large enough,...

![image](https://user-images.githubusercontent.com/84156714/193543816-1074f0d2-8e09-4462-9c5c-9afae6fb41b8.png)